app/views/modals/paymentsModal.html

```diff
@@ -37,14 +37,23 @@ <h4>
         <tbody ng-repeat="r in registration.registrants">
           <tr>
             <td>
-              <i
-                class="fa"
-                ng-class="{'fa-plus-square-o': !expandRegistrantCost[r.id], 'fa-minus-square-o': expandRegistrantCost[r.id]}"
+              <button
+                type="button"
+                class="btn-unstyled"
                 ng-show="!r.withdrawn"
                 ng-click="expandRegistrantCost[r.id] = !expandRegistrantCost[r.id]"
-              ></i>
+                aria-label="Toggle registrant cost details"
+                ng-attr-aria-describedby="registrant-name-{{r.id}}"
+              >
+                <fa-icon
+                  ng-class="{'fa-plus-square-o': !expandRegistrantCost[r.id], 'fa-minus-square-o': expandRegistrantCost[r.id]}"
```

Comment:
Optional: I believe it's possible to set up `scope: { icon: '@' }` on `fa-icon` and then watch `$scope.icon` for changes so that we could write this as. But also what you have is great and accomplished the goal of adding `aria-hidden` to all the icons.

`<fa-icon icon="{{ expandRegistrantCost[r.id] ? 'minus-square-o' : 'plus-square-o' }}" />`